.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -46,6 +46,7 @@ body:
       label: Superset version
       options:
         - master / latest-dev
+        - "4.1.0"
         - "4.0.2"
```

Comment:
You should remove 4.0.2 as 4.1 will be the supported version.

Reply:
I'll do this as part of the master merge branch